src/View.Uno/View.Uno.csproj

```diff
@@ -25,89 +26,81 @@
     <PackageReference Include="Uno.UI" Version="4.0.7" />
     <PackageReference Include="Uno.Core" Version="4.0.1" />
     <PackageReference Include="Uno.UI.Adapter.Microsoft.Extensions.Logging" Version="4.0.7" />
-	  <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.0" />
-	  <PackageReference Include="System.Reactive" Version="4.4.1" />
-	  <PackageReference Include="Uno.XamlMerge.Task">
-		  <Version>$(_Uno_XamlMerge_Task_Version)</Version>
-	  </PackageReference>
+	<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.0" />
+	<PackageReference Include="System.Reactive" Version="4.4.1" />
+	<PackageReference Include="Uno.XamlMerge.Task">
+		<Version>$(_Uno_XamlMerge_Task_Version)</Version>
+	</PackageReference>
   </ItemGroup>
 
-	<ItemGroup>
-		<XamlMergeInput Include="Controls\**\*.xaml" />
-	</ItemGroup>
```

Comment:
we will need to keep that, but with the right path (files are in the shared project now)